### PR TITLE
add: dropbox

### PR DESCRIPTION
**Pull request by chyroc**
## Screencast

save

config

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @chyroc 👋 

Thanks for taking your time to contribute 🔥 

Before going to the review state, could you update the README with a step-by-step guide on how to achieve the token – it was not possible for me to locate it 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by chyroc** [comment]
@pernielsentikaer hi, i fix my pr, please review it again, thanks.

**Review by pernielsentikaer — COMMENTED** [review]
Hi @chyroc 👋 

We might have a problem, I'm not sure how it handles when it's downloading a large file and the extension window loses focus, when that happens, everything will be terminated.

This is not ideal, since we can't force end-users to have the extension open at all times in order to get the file downloaded.

I'm not able to test it out myself yet since I keep getting 401 with wrong permissions, but if it's the case, then we have to set it on hold until it's possible to have background processes.

Suggested feedback was added before above was written, I'm letting it be 😄 

Feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/dropbox/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "dropbox",
+  "title": "dropbox",
+  "description": "dropbox extensions",
```
[the remark this answers is not in the document]
Comment (chyroc) [reply extensions/dropbox/package.json]: hi, this extension is for all dropbox feature, save url is one of features(example: view file, upload file, save file, ...)

[the remark this answers is not in the document]
Comment (chyroc) [reply extensions/dropbox/package.json]: i change it to `Dropbox Extensions`

**Comment by chyroc** [comment]
@pernielsentikaer hi, i fix this pr, please review it again , thanks.

**Comment by pernielsentikaer** [comment]
Hi @chyroc 👋 

I'm not sure why it didn't work the first time, but now I'm able to test it out.

It seems to do the job as expected with smaller files, however, if I download larger files and the extension looses focus, it dies.

As stated earlier, we don't have support for background processes, until then we have to set it on hold since we don't want bad user experience.

Feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by chyroc** [comment]
@pernielsentikaer hi:

If I use this api ( https://developers.raycast.com/api-reference/storage ) to store and reload data, is there no need for a long running background process?
And, you can do a job list function locally.

If you think this note is ok I will revise my pr

**Comment by pernielsentikaer** [comment]
Hi @chyroc 👋 

I'm not sure what you have in mind, are you thinking about splitting it up in a list with jobs and a command to add? I would like to hear a bit more of what you have in your mind 😄

**Comment by fe9lix** [comment]
Hey guys, I'm closing this PR because of inactivity. Feel free to reopen it again later and address the feedback.

**Pull request closed without merging**